pkg/storer/migration/step_05.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2024 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package migration
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ethersphere/bee/pkg/log"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/storer/internal/upload"
+)
+
+// step_05 is a migration step that removes all upload items from the store.
+func step_05(st storage.BatchedStore) error {
+	logger := log.NewLogger("migration-step-05", log.WithSink(os.Stdout))
+	logger.Info("start removing upload items")
+
+	var itemsToDelete []storage.Item
```

Comment:
there is a chance you might blow up the memory.
i would use a channel here so that one worker feeds the items, the other picks them up and deletes from storage